Formula/rex.rb

```diff
@@ -205,10 +227,18 @@ class Rex < Formula
     sha256 "771f7d02abd1ded94d9e37d3f66e795c8d2026d04defbeb5b679ca058116bbf3"
   end
 
+  resource "inc::latest" do
+    url "http://search.cpan.org/CPAN/authors/id/D/DA/DAGOLDEN/inc-latest-0.500.tar.gz"
+    sha256 "daa905f363c6a748deb7c408473870563fcac79b9e3e95b26e130a4a8dc3c611"
+  end
+
   def install
     ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
     ENV.prepend_path "PERL5LIB", libexec/"lib"
 
+    # Spare Perl v5.23+ users a long-winded gripe from JSON::XS module's author.
```

Comment:
His FAQ actually states that if your build or test-suite passes, there's nothing to worry about:

> **"Does this mean the module will not work?"**  
> "No, in most cases, it will work. Currently only Coro will definitely not work with 5.22.0, while most or even all of my other modules will probably still work.  
>  
> **So as long as everything seems to work (and/or the testsuite runs successfully), you are most likely in business and can ignore the warning.**

And then there's this:
> What it does mean, and I am quite serious about this, is that I don't want to hear about problems with perl 5.22.0 or any later release - these are simply not supported. They might work perfectly fine, but you are definitely on your own.